casper/src/test/resources/PoSTest.rho

```diff
@@ -71,20 +69,6 @@ match (
             }
           } |
 
-          contract test_close_block_finishes(rhoSpec, _, ackCh) = {
```

Comment:
I don't agree with this removal. This is a smoke test for closeBlock. If it fails you know where to look for a class of problems. Without this test you have a much bigger area to search for problems when the tests hang